airbyte-integrations/connectors/destination-duckdb/destination_duckdb/destination.py

```diff
@@ -14,14 +14,7 @@
 import duckdb
```

Comment:
This is not related `badly formatted` file, I've just included it to fix the `master`.